config/container_resolvers_conf.xml.sample

```diff
@@ -8,13 +8,13 @@
 
   <!-- <cached_mulled /> -->
   <!-- cached_mulled: resolves container URI through bioconda to mulled
-       automatic mapping, preferring cached containers in the accessible docker
+       automatic mapping, preferring cached images in the accessible docker
```

Comment:
yeah, was hesitating about this. Galaxy doesn't do any `container` things at all, I think we pretty consistently use image where sometimes that's the wrong word :(.

Reply:
Ok, feel free to strip that commit.

Reply:
nah, at least here it's correct.